Ochowo/iReporter
# [#62167697] update sigin.html and admin.html

What does this PR do?
- Make dropdown on admin page responsive and show the image on the sign in page.

**Description of Task to be completed**
- Fix the drop-down and ensure the image on the sign-in page is showing

**How should this be tested manually**
- After cloning the repo, cd into it and open the admin.html template or the sigin.html template to see the changes.

**Relevant Pivotal Stories**

- [#62167697](https://www.pivotaltracker.com/story/show/162167697)
